mopidy_gmusic/library.py

```diff
@@ -311,14 +313,30 @@ def sorter(track):
     def refresh(self, uri=None):
         self.tracks = {}
         self.albums = {}
+        self.albums2 = {}
```

Comment:
Could this be named something a bit more descriptive? It's always nice to be able to look at a variable and get a bit of context about how it's used.

Reply:
Can do, no problem